### DISCUSSION TITLE
The feed is terible

### DISCUSSION THREAD

**Discussion by PurpleBored** [community/community#58441] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

No one asked for it everyone hates it waste of space like no one wants for github to become another algorithm driven "for you feed" 
Please just get rid of it

**Comment by mrkrabs206** [comment]
Yes

**Comment by branch-tim** [comment]
Yes, "for you" is an incredible waste of space.

**Reply by mrkrabs206** [reply]
YOURE RIGHT!

**Comment by SammyCola** [comment]
"For you" should stay with social media where it belongs.

**Reply by PurpleBored** [reply]
fr

**Reply by mrkrabs206** [reply]
fr fr fr fr fr 💀💀💀

**Comment by alx3dev** [comment]
Was the best UI, now it's the worst one.

**Comment by roelkok** [comment]
It is very distracting for me. I often find myself logging into Github to accomplish a task, but finding myself checking out some trending new repo instead.

GitHub is supposed to be a productivity tool. The less time spent in GitHub should be considered a good thing. The engagement driven fyp doesn't belong here. It feels like it has been thought up by some growth hacker focusing on a time-spent-on-site metric and does not stem from what users actually want, ie accomplishing their task as fast as possible.

**Reply by PurpleBored** [reply]
They should really make it opt in

**Comment by anne-gropler** [comment]
The "For You" page is the reason why I enter github only via my bookmarks directly linking to the repository I'm currently working on.

**Comment by GregDomzalski** [comment]
I come here to do work. To be productive. Not to try and "discover cool new projects". That's what I have HackerNews and web search for.

I want my projects, my organizations, my tasks _front and center_. That is what a **dashboard** is _supposed_ to be!

Stop social media-ing everything, _please_.

**Reply by GregDomzalski** [reply]
I guess I would also add that I'm not completely opposed to a "Discover" tab or page or something. That's where this sort of thing belongs. In its own little section. Not as the primary function of my own dashboard.

**Reply by David-at-TechCXO** [reply]
I'm with @GregDomzalski here. THIS IS NOT SOCIAL MEDIA. I'm here to get work done.

**Comment by ozeze12** [comment]
Nicer bro

**Comment by philkunz** [comment]
Its a waste of energy. Makes GitHub a social media trashcan. I want to know whats going on in my organizations, not whats going on with a repo I starred at some point.

**Comment by hamidrezamd1380** [comment]
https://realit.ir/

**Comment by CrzyMan** [comment, marked as answer]
# Is Github going to sell my data?
Github is a **productivity** website, not a **social media** website. Those two types of websites have nearly antithetical goals and intentions to one another. You CANNOT be both.
Github is trying to integrate more features that are ONLY relevant to social media sites.
The logical conclusion is that it is trying to forgo it's usefulness in favor of being ~~wasteful~~ entertaining.
Moving forward, we can anticipate that they will actively sabotage our productivity if they think they'll make a buck off it.

## Social sites sell your data
Social sites are all about getting you to stay on the site as long as possible so they can farm and sell as much data on you as possible.
It is an incredibly profitable and unethical business model. My personal opinion is that Meta/Google/Amazon/Tencent/ByteDance/Microsoft should be fined to hell and back for profiting massively by completely disregarding the privacy of literally billions of people.

## Suggestions will become glorified ads
It starts off simple enough, "you can pay to promote your repo". It's not an "ad", no, never. It's a promotion! We're just helping small communities grow!

Then it gets insidious because marketers get involved and are used to having access to personal information on their targets.

So you start selling "premium promotions" where you can target ~~victims~~ demographics based on their interests...

*and what 3rd party services they log into with Github...*

At that point, all suggestions are useless because they are ALL ads.
Ad blockers start blocking them.
Now no one sees any suggestions at all.
Just as God intended.

## Feeds are an accessibility issue
Feeds (especially infinite ones) are well known for exploiting psychology to trap individuals. 
To lull them into a false sense of security and then never let them leave, at least not until they've been milked for as much attention as they can.

Like seriously, I'm ADHD. The UI has gotten so cluttered that I can't obviously tell what's relevant to me.
I impulsively click something I didn't intend to.
I go down rabbit holes I didn't intend to.
I travel much further down those rabbit holes than I intend to.
I spend a **LOT LESS TIME WORKING** than I intend to.

That wouldn't be a problem if I was casually browsing Github because I was bored...
but I'm not here because I'm bored... **I'm here because I'm working**!

## Conclusion: Github is actively rotting but is salvageable
The feed isn't the problem. It's the implications of them NOT running away from the idea.
Feed -> social site -> makes money selling user data/ads -> need people to not ignore ads -> makes site harder to use

The answer is easy: **Make it opt-in**

You don't even have to get rid of it. But you can absolutely get lost if you try to make me use it.

Sorry for the TEDx talk, but I'm just thoroughly convinced that they are afraid we'd crucify them if they were honest with why they did this.

Edit: Typo

**Reply by CrzyMan** [reply]
@PurpleBored lol, I guess my ADHD forcing me to hyperfocus this for 2 hours wasn't for naught XD I also ended up writing a TamperMonkey (plugin) script that finds and deletes elements from whatever Github page I'm viewing. Mostly targeting the entire sidebar or specific feed related elements. Turns out, I don't need or want like 50% of what I'm shown by Github.

Searching through some other threads, I am just resorting to using bookmarks as my only way of interacting with Github. If their UI can't give me the means to navigate it well, I'll just make the means for myself.

Thanks to my therapist and some significant strides in personal acceptance after my adult diagnosis of ADHD, self-accommodations are just a normal thing to me now. But I can't help but feel like I shouldn't NEED them to use a productivity tool.

**Comment by levicki-printec** [comment]
I do not have ADHD (at least that I know of), but I wholeheartedly agree that Feeds, Follows, Recommendations, Suggestions, For You, etc **belong to social media platforms, not to ANY kind of working environment**.

This isn't Twitter, Facebook, Instagram, TikTok, or LinkedIn and I don't want to "engage" with GitHub in the same way I would have to engage with those social media platforms.

Furthermore, Microsoft keeps forgetting that both Visual Studio and GitHub are also used by a large number of individual developers and focuses on designing everything for and around teams, going as far as to either nag people to use team features (best case), or actively force those features on everyone (worst case).

Then there's the matter of not giving me as a user any choice. For example, someone has chosen to follow my personal GitHub account. Nobody asked for my consent to be followed. I understand that I cannot stop someone determined from manually stalking me, but why make it easy for them by allowing them to subscribe to my every action? It is creepy and unnecessary, especially in a work environment.

Finally, please stop with "For You", "Recommendations", "Suggestions", etc sections everywhere &mdash; **you don't know what I want better than I do, and the fact that you believe otherwise and insist on shoving that down my throat just shows how arrogant and rude you are**. Every time I see that I feel insulted and bullied. JUST. STOP. IT. ALREADY.

Thank you for your time.

**Comment by tomoverlund** [comment]
Ooh, a new feed! Maybe it will show me things from projects I have actually interacted with! Oh, what's that, it's just random crap? Oh, you want me to have to actually follow project instead of just showing what makes sense? Oh, okay... yeah, no.

**Comment by Debaditya-Som** [comment]
GitHub functions as a productivity-oriented platform rather than a social media platform, and these two categories of websites fundamentally oppose each other in their purposes. It is not feasible for a website to simultaneously serve both roles.

However, GitHub has been incorporating additional features that are specifically designed for social media sites. This indicates a shift towards prioritizing entertainment over its core usefulness.

As we look ahead, there is a concern that GitHub might compromise its productivity-focused aspects in pursuit of profit, even if it means hindering users' productivity.

**Comment by brett-sprad** [comment]
I agree, I feel the new releases of Github are just looking more & complex in the UI. The main reason why I love(d) Github was because it had such a friendly UI. But now I feel they are just complicating the UI. There's so many "new" things, that really aren't helpful and take you away from the core reasons / core workflows that one needs to perform during the day.

**Comment by 0x427279** [comment]
I'm trying to focus on my code, and this is just distracting and completely irrelevant and I'm wasting time continuously clicking on 'less of this' to no avail. Please give me the ability to disable this annoying feature.

I'm currently trialing github with a view to moving from our internally hosted gitlab instance to cloud hosted github. Can you guess how that's going when I'm feeling aggravated by this 'feature'?

**Comment by andrewpmartinez** [comment]
Its garbage. I  want a list of things I need to do. 

- Open PRs I've been mentioned on directly or via a group.
-  New comments that tag me or are a reply to me

And that is pretty much it. The home page redesign is useless to me.

**Comment by CliffBramlett** [comment]
TL;DR: **Keep shoving this down our throats and we'll make somewhere else to go. We're devs, after all.**  
  
I was hoping that when MS bought this platform that they would just leave it alone, but was skeptical considering MS history. I am sad that skepticism is proven right.  
  
Social media platforms are failing. **People are tired of being manipulated.** How would MS managers feel if the first 10 min of EVERY meeting you have you must play a video advertising my app, and AI on cameras watches to make certain you watch, and your pay is docked if you don't? Your engagement tripe is being shoved in our faces and while you can't dock (most of) our pay, it feels like you are since you're taking time out of our productive hours to deal with your alerts.  
  
Go back to the concepts that brought all us devs here, and convinced you (Microsoft) to buy GitHub in the first place.

**Comment by NitinIsFit** [comment]
Well, the feed was made so we could find repositories which we want to, like I made so many discord bots, so GitHub takes data from what kind of threads I am joined in, what kind of repositories I make and what I title them, what tags I add in them and also what kind of other repositories have similar title, tags and code files in them, GitHub takes all of it and gives us the feed

But to be honest, this feed system is always boring, it doesn't have variety like YouTube.
Like I cannot set filter to "today's trending/most starred", "this week's most started", "this past month's most starred", "this past year's most starred", and they do not even have a search bar so I could search in feeds page.

**Reply by CrzyMan** [reply]
The problem is that's only one thing it was made to do. It was probably also made to track your interest in the hopes that GitHub would eventually be able to **sell you** to the highest bidder. Whatever repository promoter has the most money is who you'd see the most. They are probably trying to break into the incredibly lucrative and unethical business of being an ad broker.

Also, it wouldn't be that bad if we could just avoid it. But we can't. Not even if you wanted to. You have to go out of your way to get even close.
If you go to [GitHub.com](https://github.com/) while logged in, >80% of the screen is devoted to these new suggestion features. And you CANNOT turn them off.

To quote you
> this feed system is always boring,

Correct. And it always will be. There is literally no possible future where Github is still useful AND the feed is also filled with a velocity and variety that could feed our interest.

There is no hope for the feed. It is a lost cause. It is worse than worthless. It is actively reducing the value of this site.

This is a productivity website. Social media sites are quite explicitly, by design, productivity blockers.

If the feed is ever good, it means that Github is no longer useful.

**Reply by clort81** [reply]
> Well, the feed was made so we could find repositories which we want to, 
SEARCH
DONT
SPAM
ME
WITH
YOUR
S,...
